### ISSUE TITLE
Error executing fix on maps: "encode 'DataVersion' error: unsupported type int"

### ISSUE THREAD

**Issue by bierdosenhalter** [nothub/mapart#3]
I'm experiencing an "encode 'DataVersion' error: unsupported type int" while attempting to apply the fix on a 1.12.2 map. It appears that changing the type from int to int32 resolved the issue.

**Comment by nothub** [comment]
I can reproduce this:
```
$ go run . fix samples/maps/1.12.2
encode "DataVersion" error: unsupport type int
encode "DataVersion" error: unsupport type int
```

**Comment by nothub** [comment]
Fixed with c7d49feb15cff82dd63c739c5cb99cccd54e2ed2

Thanks for the report!
